javascript-view-swapping/main.js

```diff
@@ -0,0 +1,18 @@
+const $container = document.querySelector('.tab-container');
+const $tab = document.querySelectorAll('.tab');
+const $view = document.querySelectorAll('.view');
+
+$container.addEventListener('click', () => {
+     if (event.target.matches('.tab')) {
+      for (let i = 0; i < $tab.length; i++) {
+        $tab[i].className = 'tab'
+        event.target.className = 'tab active';
+
+        const $dataView = event.target.getAttribute('data-view');
+        $view[i].className = 'view hidden'
```

Comment:
This should be apart of your `else` statement in the `if` statement down below. We only want this to run if the conditional in the `if` statement below is false